policy-staging.json

```diff
@@ -0,0 +1,43 @@
+{
+  "instance_min_count": 1,
+  "instance_max_count": 3,
+  "scaling_rules": [
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 10,
+      "operator": "<",
+      "cool_down_secs": 60,
+      "adjustment": "-1"
```

Comment:
Paas docs suggested fairly gradual and often scaling. Which i quite like the idea of